### DISCUSSION TITLE
Z10 Pro - Getting `cat: can't open '/etc/os_release': No such file or directory` instead of password prompt

### DISCUSSION THREAD

**Discussion by jacobwod** [Hypfer/Valetudo#1768] [Q&A / Support] [answered]
Hi! I recently tried to help with rooting a new Dreame Z10 Pro using the UART method as described in FAQ/docs. The robot boots and the UART connection works with terminal output showing log messages. 

When the Wifi reset method is attempted, instead of getting the expected prompt for new password, all that shows up is this:
```
cat: can't open '/etc/os_release': No such file or directory
```

The kernel and firmware version is `4.1.7_1099`. 

Has anyone seen this "error" message when attempting to root your devices and, if yes, how did you mange to overcome? Is it [related to this](https://github.com/Hypfer/Valetudo/discussions/1556#discussioncomment-3441744) even though it's a different robot and does the same procedure apply? 

Thanks!

**Comment by bugwelle** [comment, marked as answer]
Hi, please see https://builder.dontvacuum.me/dreame/faq.txt and look for the Dreame User Group Telegram channel. You will likely get help there 😄
